test/integration/serverless/snapshots/log-csharp

```diff
@@ -1,170 +1,182 @@
 [
   {
     "message": {
-      "message": "START RequestId: XXX Version: $LATEST",
+      "message": "END RequestId: XXX",
```

Comment:
The order of the log messages has been updated. It is now in alphabetical order by the message contents rather than in numberical order based on the timestamp.